src/popup/router/components/InputAmount.vue

```diff
@@ -5,23 +5,20 @@
       min_value_exclusive: 0,
       ...+balance.minus(fee) > 0 ? { max_value: max } : {},
       enough_ae: fee.toString(),
-      not_token: noToken,
       ...validation,
     }"
     name="amount"
-    :error="$attrs.error || errors.has('amount')"
-    :error-message="$attrs['error-message'] || errors.first('amount')"
+    :message="$attrs['message'] || errors.first('amount')"
     class="input-amount"
     type="number"
     v-bind="$attrs"
     placeholder="0.00"
     :label="$attrs.label || $t('pages.tipPage.amountLabel')"
     @input="$emit('input', $event)"
   >
-    <template slot="right">
-      <span class="token">{{ selectedToken ? selectedToken.symbol : 'AE' }}</span>
+    <template #after>
+      <span class="token">{{ 'AE' }}</span>
```

Comment:
```suggestion
      <span class="token">AE</span>
```

No need to use moustache here.